packages/shylu/shylu_dd/frosch/src/SchwarzOperators/FROSch_AlgebraicOverlappingOperator_decl.hpp

```diff
@@ -83,15 +86,17 @@ namespace FROSch {
             FROSCH_ASSERT(false,"AlgebraicOverlappingOperator cannot be built without input parameters.");
         };
 
-        int initialize(int overlap,
+        virtual int initialize(int overlap,
                        ConstXMapPtr repeatedMap = null);
 
-        int compute();
+        virtual int compute();
 
-        void describe(FancyOStream &out,
+        virtual void describe(FancyOStream &out,
                       const EVerbosityLevel verbLevel=Describable::verbLevel_default) const;
 
-        string description() const;
+        virtual string description() const;
+
+        virtual void printParameterDescription() const;
```

Comment:
This is not strictly needed but it was previously directly contained in the initialize method of the AlgebraicOverlappingOperator and a rather large codeblock. So I extracted it to separate two functionalities: 1. Initializing the operator, 2. providing output for the user.
In addition it allowed me to simply add the new parameters in the HarmonicOverlappingOperator to the output. This would not have been possible without large code duplication before.

``` C++
void HarmonicOverlappingOperator<SC,LO,GO,NO>::printParameterDescription() const
    {
        AlgebraicOverlappingOperator<SC,LO,GO,NO>::printParameterDescription();
        cout
            << "\n" << setw(FROSCH_OUTPUT_INDENT) << " "
            << setw(89) << "========================================================================================="
            << "\n" << setw(FROSCH_OUTPUT_INDENT) << " "
            << "| " << left << setw(41) << "HarmonicOnOverlap" << right
            << " | " << setw(41) << this->ParameterList_->get("HarmonicOnOverlap",false)
            << " |"
```